en/releases/stable_v3.2_long.md

```diff
@@ -179,38 +179,54 @@ There is an armed/disarmed indicator in the toolbar. You can click it to arm/dis
   * Goto Location
```

Comment:
Ignore all changes to this, they are just typos and minor fixes. Some of this will be deleted when I know this PR is good.